### ISSUE TITLE
SRGAN model not working

### ISSUE THREAD

**Issue by senyosimpson** [senyosimpson/sres#1]
SRGAN model outputs images that are off-colour. No real idea why at this point

**Comment by senyosimpson** [comment]
Did not normalise images in the format expected by pretrained PyTorch models.
